bones/codeBone.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+from viur.core.bones import baseBone
+
+class codeBone(baseBone):
+	'''
+		The codebone does not check and excaped the entered content.
+		This can lead to security problems.
+		Therefore, like the basebone, use this bone with caution.
```

Comment:
Please change this to:
> Likeweise the baseBone, the codeBone does not check or escape the provided content.
> This might lead to security problems, but is required under some circumstances.
> codeBone just specializes its content more specifically so it can be easily adapted.
> Use the dot-notation like "code.markdown" or similar to describe subsequent types.